client/finality-grandpa/src/import.rs

```diff
@@ -431,6 +431,12 @@ where
 				e
 			))
 		})?;
+
+		let hash = self
+			.inner
+			.expect_block_hash_from_id(id)
+			.map_err(|e| ConsensusError::Other(e.into()))?;
```

Comment:
```suggestion
```

Please make this function take `Block::Hash` directly.

Reply:
done in [11c7013](https://github.com/paritytech/substrate/pull/12510/commits/11c7013119e839560327b126f2a64be00edc3cba)